ajax/libs/flowplayer/package.json

```diff
@@ -16,5 +16,18 @@
     "type": "git",
     "url": "https://github.com/flowplayer/flowplayer"
   },
-  "license": "GPLv3"
+  "author": {
+    "name": "Niklas Närhinen",
+    "url": "https://github.com/nnarhinen"
+  },
+  "license": "GPL-3.0",
+  "npmName": "flowplayer",
```

Comment:
This lib has 27 releases on GitHub repo but only 6 on npm. Is there any reason to use npm auto-update instead of git auto-update?